src/components/contextual/pages/pool/PoolContractDetails.vue

```diff
@@ -26,11 +26,11 @@ const { t } = useI18n();
 const { explorerLinks: explorer } = useWeb3();
 
 function formSwapFeesHint(owner: string): string {
-  if (owner === POOLS.ZeroAddress) {
+  if (owner === configService.network.pools.ZeroAddress) {
```

Comment:
Now I'm thinking maybe it should be `POOLS` to not conflict with local declarations of `pools`, but that makes it seem like a constant which it kind of is but also isn't because it's per network config.

Reply:
I've adjusted everything now. I like the importing pools from config, it's consistent to how we import the overall config object and we could use a similar pattern for tokens or other commonly used config settings.

Reply:
Great, the class was just an example. I usually prefer functions over classes so I prefer your solution 👌